docs/installation/basic.md

```diff
@@ -3,10 +3,12 @@
 ## Dependencies required to run Orga Server
 
 * Python 3.7
-* Postgres
+* PostgreSQL
+* Redis
+
+
 ```sh
-sudo apt-get update
-sudo apt-get install postgresql postgresql-contrib
+xargs -a deb-packages.txt sudo apt install
```

Comment:
The repo is not cloned at this point. People will copy paste the command and be greeted with an error

Reply:
Then I move this command down to after "cloning".

Reply:
This part is already after "cloning".